src/views/Home/components/ChapelProgress/index.js

```diff
@@ -45,16 +45,16 @@ export default class ChapelProgress extends Component {
     } else {
       content = <Doughnut data={data} />;
       if (current === 1) {
-        subheader = `${current} CL&W`;
+        subheader = `${current} CL&W credits`;
       } else {
-        subheader = `${current} CL&W's`;
+        subheader = `${current} CL&W credit's`;
```

Comment:
There should be no apostrophe ("credits" not "credit's") because it is not possessive.